public/assets/stylesheets/bootstrap.min.css

```diff
@@ -1,7 +1 @@
-/*!
```

Comment:
I was experiencing a strange glitch where the handlebars files wouldn't use the CSS files. Someone on StackOverflow suggested to not start a CSS file with a comment.